### ISSUE TITLE
During training,I found the loss_classifier is much higher than other loss. Is it because of the pretrain model or the backbone net？

### ISSUE THREAD

**Issue by a954217436** [facebookresearch/maskrcnn-benchmark#1030]
## ❓ Questions and Help
Is there methods I can do to decrease the loss_classifier? Or should I change another backbone？I'm using Resnet now......
Thanks for answering my question. I wish you have a nice day!
